ci/runner.js

```diff
@@ -0,0 +1,149 @@
+/*
```

Comment:
We should probably call this something more specific, like `phantomjs.js` -- otherwise to someone exploring the repo, from its name it sounds like a more central part of our CI setup.